tests/Validator/FormatValidatorTest.php

```diff
@@ -81,14 +68,7 @@ public function testValidateOnlyReferenceIsAllowedIfNotFormats()
         $gallery->expects($this->once())->method('getDefaultFormat')->will($this->returnValue('format_that_is_not_reference'));
         $gallery->expects($this->once())->method('getContext')->will($this->returnValue('test'));
 
-        // Prefer the Symfony 2.5+ API if available
-        if (class_exists('Symfony\Component\Validator\Context\ExecutionContext')) {
-            $contextClass = 'Symfony\Component\Validator\Context\ExecutionContext';
-        } else {
-            $contextClass = 'Symfony\Component\Validator\ExecutionContext';
-        }
-
-        $context = $this->getMockBuilder($contextClass)
+        $context = $this->getMockBuilder(ExecutionContext::class)
```

Comment:
please use `createMock()` whenever possible

Reply:
Will try, but only on the code I changed, because this is not the objective of the PR

Reply:
Done!